ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-ref.yaml

```diff
@@ -38,6 +38,9 @@ ref:
   - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
     default: "SingleReplica"
     documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+  - name: OLM_CATALOG_PLACEMENT
+    default: "guest"
+    documentation: "The OLM catalog placement."
```

Comment:
Could you specify the other available options and explain about them?

Reply:
added other options